### PR TITLE
[ENDPOINT] GET /categories/:id

### PR DESCRIPTION
**Pull request by AlexAc0226**
AS admin user
I WANT to get information from an individual category
To be able to manage it

Criteria of acceptance:
GET a /categories/:id . It will show all the fields of a category based on the id sent by parameter. In the event that it does not exist, it will return an error with a 404 status code.

Evidence route:
image

service/categories:
image

controllers/categories:
image

### CODE REVIEW

File: services/categories.js [hunk 1]
```diff
@@ -1,13 +1,20 @@
 const { ErrorObject } = require('../helpers/error')
 const { Category } = require('../database/models')
 
-// example of a service
-exports.getCategories = async () => {
+exports.getCategories = async ({ id }) => {
```
Comment (gabodmc) [inline services/categories.js]: getCategoryById en el servicio.

**Review by gabodmc — CHANGES_REQUESTED** [review]
El pr esta bien encaminado. Lo que seria la evidencia deberia mostrar imagenes del postman listando la categoria en caso de que funcione, y los casos de error (no encontrada/sin parametros9

**Review by gabodmc — COMMENTED** [review]
Bien resuelto. Recordar poner la evidencia del postman y no solo la respuesta "OK"

**Review by Franckfer — APPROVED** [review]

**Review by afrivera — APPROVED** [review]
Buen PR

**Review by gabodmc — APPROVED** [review]

**Comment by AlexAc0226** [comment]
Eviden: route result GET categories/:id
image

**Pull request merged**